terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 1:  7762 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-9-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 2:  7764 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-9-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 3:  7766 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-9-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 4:  7768 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-9-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 5:  7770 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-9-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 6:  7772 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-10-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 7:  7774 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-10-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 8:  7776 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-10-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 9:  7778 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-10-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 10:  7780 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-10-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 11:  7782 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-11-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 12:  7784 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-11-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 13:  7786 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-11-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 14:  7788 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-11-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 15:  7790 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-11-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 16:  7792 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-12-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 17:  7794 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-12-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 18:  7796 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-12-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 19:  7798 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-12-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 20:  7800 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-12-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 21:  7802 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-13-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 22:  7804 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-13-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 23:  7806 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-13-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 24:  7808 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-13-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 25:  7810 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-13-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 26:  7812 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-14-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 27:  7814 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-14-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 28:  7816 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-14-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 29:  7818 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-14-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 30:  7820 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-14-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 31:  7822 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-15-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 32:  7824 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-15-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 33:  7826 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-15-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 34:  7828 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-15-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 35:  7830 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-15-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 36:  7832 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-16-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 37:  7834 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-16-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 38:  7836 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-16-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 39:  7838 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-16-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 40:  7840 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-16-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 41:  7842 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-17-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 42:  7844 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-17-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 43:  7846 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-17-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 44:  7848 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-17-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 45:  7850 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-17-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 46:  7852 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-18-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 47:  7854 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-18-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 48:  7856 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-18-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 49:  7858 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-18-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 50:  7860 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-18-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 51:  7862 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-19-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 52:  7864 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-19-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 53:  7866 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-19-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 54:  7868 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-19-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 55:  7870 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-19-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 56:  7872 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-20-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 57:  7874 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-20-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 58:  7876 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-20-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 59:  7878 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-20-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 60:  7880 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-20-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 61:  7882 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-21-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 62:  7884 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-21-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 63:  7886 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-21-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 64:  7888 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-21-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 65:  7890 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-21-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 66:  7892 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-1-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 67:  7894 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-8-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 68:  7896 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-3-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 69:  7898 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-5-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 70:  7900 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-4-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 71:  7902 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-2-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 72:  7904 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-6-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 73:  7906 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-1-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 74:  7908 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-6-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 75:  7910 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-4-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 76:  7912 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-2-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 77:  7914 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-3-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 78:  7916 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-6-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 79:  7918 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-1-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 80:  7920 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-5-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 81:  7922 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-2-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 82:  7924 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-1-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 83:  7926 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-7-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 84:  7928 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-6-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 85:  7930 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-3-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 86:  7932 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-8-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 87:  7934 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-4-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 88:  7936 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-2-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 89:  7938 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-5-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 90:  7940 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-3-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 91:  7942 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-7-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 92:  7944 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-1-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 93:  7946 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-8-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 94:  7948 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-2-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 95:  7950 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-6-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 96:  7952 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-7-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 97:  7954 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-5-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 98:  7956 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-8-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 99:  7958 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-3-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 100:  7960 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-5-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/rp/17.neglite.program'
./scripts/neg/ijcai19/rp-val/rp-NEGLITE-1-3-d-val.sh: line 101:  7962 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/rp/domain.pddl domains/neg/rp/rp-8-neg-a.pddl $1
